src/orchestration/Orchestration.ts

```diff
@@ -294,6 +297,14 @@ export class Orchestration {
         this.pluginManager.record(JS_ERROR_EVENT_PLUGIN_ID, error);
     }
 
+    /**
+     * Update DOM plugin to record the (additional) provided DOM events.
+     * @param pluginConfig Target DOM events.
+     */
+    public registerDomEvents(pluginConfig: object) {
+        this.pluginManager.updatePlugin(DOM_EVENT_PLUGIN_ID, pluginConfig);
```

Comment:
```suggestion
    public registerDomEvents(events: TargetDomEvent[]) {
        this.pluginManager.updatePlugin(DOM_EVENT_PLUGIN_ID, { events });
```

Reply:
As a user, I would expect `registerDomEvents` to take a list of events. We can then wrap it as a config object.

Reply:
> We can then wrap it as a config object.
Or leave it as-is if the parameter for `DomEventPlugin.update` is an array of events.

Reply:
Updated as suggested as well as in CommandQueue.ts to take in object of type any.